Library/Homebrew/cmd/tap.rb

```diff
@@ -63,10 +69,11 @@ def each_tap
     end
   end
 
-  def tap_args(tap_name=ARGV.named.first)
+  def tap_args()
+    tap_name, clone_target = ARGV.named[0..1]
```

Comment:
That would require more branching. This struck me as an easier, simpler solution. If some method is going to change, why *not* this one?